### PR TITLE
Visit cancellation receipt contains VO number for reference.

### PR DESCRIPTION
**Pull request by jasiek**
[#86310816]

### CODE REVIEW

File: app/views/prison_mailer/booking_cancellation_receipt_email.text.erb [hunk 1]
```diff
@@ -2,6 +2,10 @@ Dear staff,
 
 <%= @visit.visitors.first.full_name %> has cancelled the visit to <%= @visit.prisoner.full_name %> (<%= @visit.prisoner.number %>) on <%= visit.slots.first.date %> <%= visit.slots.first.times %>.
 
+<% if @visit.vo_number %>
+Reference number: <%= @visit.vo_number %>
+<% end %>
+
```
Comment (matharden) [inline app/views/prison_mailer/booking_cancellation_receipt_email.text.erb]: Please could you move this space inside the `if` block so it doesn't create a double space when there is no reference number?

File: spec/mailers/prison_mailer_spec.rb [hunk 2]
```diff
@@ -94,12 +94,14 @@
     end
 
     it "sends an cancellation notification to a prison so it can be removed from NOMIS" do
+      sample_visit.vo_number = '87654321'
```
Comment (matharden) [inline spec/mailers/prison_mailer_spec.rb]: Might be good if this was moved into the :sample_visit so the VO number appears in mailer previews.

Comment (jasiek) [inline spec/mailers/prison_mailer_spec.rb]: Yeah, good idea.

File: app/views/prison_mailer/booking_cancellation_receipt_email.text.erb [hunk 3]
```diff
@@ -2,6 +2,9 @@ Dear staff,
 
 <%= @visit.visitors.first.full_name %> has cancelled the visit to <%= @visit.prisoner.full_name %> (<%= @visit.prisoner.number %>) on <%= visit.slots.first.date %> <%= visit.slots.first.times %>.
 
+<% if @visit.vo_number %>
+Reference number: <%= @visit.vo_number %>
+<% end %>
```
Comment (matharden) [inline app/views/prison_mailer/booking_cancellation_receipt_email.text.erb]: Needs a line break here

**Pull request merged**